apps/generator/docs/generator-template.md

```diff
@@ -25,59 +25,79 @@ Before you begin, make sure you have the following set up:
->  
+>
+> This starts an Eclipse Mosquitto broker locally on your machine, listening on port 1883.
+>
 > If you don’t want to use Docker, you can install Mosquitto manually. Follow the [official installation guide](https://mosquitto.org/download/) for your operating system.
 
 ## Background context
 
 There is a list of [community maintained templates](https://www.asyncapi.com/docs/tools/generator/template#generator-templates-list), but what if you do not find what you need? In that case, you'll create a user-defined template that generates custom output from the generator.
 Before you create the template, you'll need to have an [AsyncAPI document](https://www.asyncapi.com/docs/tools/generator/asyncapi-document) that defines the properties you want to use in your template to test against. In this tutorial, you'll use the following template saved in the **test/fixtures/asyncapi.yml** file in your template project directory.
 
-``` yml
+```yml
 
-asyncapi: 2.6.0
 
+asyncapi: 3.0.0
 info:
   title: Temperature Service
   version: 1.0.0
   description: This service is in charge of processing all the events related to temperature.
 
 servers:
   dev:
-    url: test.mosquitto.org #in case you're using local mosquitto instance, change this value to localhost.
+    host: test.mosquitto.org
     protocol: mqtt
 
 channels:
-  temperature/changed:
-    description: Updates the bedroom temperature in the database when the temperatures drops or goes up.
-    publish:
-      operationId: temperatureChange
-      message:
-        description: Message that is being sent when the temperature in the bedroom changes.
-        payload:
-          type: object
-          additionalProperties: false
-          properties:
-            temperatureId:
-              type: string
+  temperatureDroppedChannel:
+    address: temperature/dropped
+    messages:
+      temperatureDropMessage:
+        $ref: '#/components/messages/temperatureDropMessage'
+
+  temperatureRisenChannel:
+    address: temperature/risen
+    messages:
+      temperatureRiseMessage:
+        $ref: '#/components/messages/temperatureRiseMessage'
+
+operations:
+  temperatureDrop:
+    action: send
+    channel:
+      $ref: '#/channels/temperatureDroppedChannel'
+    messages:
+      - $ref: '#/channels/temperatureDroppedChannel/messages/temperatureDropMessage'
+  temperatureRise:
+    action: send
+    channel:
+      $ref: '#/channels/temperatureRisenChannel'
+    messages:
+      - $ref: '#/channels/temperatureRisenChannel/messages/temperatureRiseMessage'
+
 components:
+  messages:
+    temperatureDropMessage:
+      payload:
+        $ref: '#/components/schemas/temperatureId'
+    temperatureRiseMessage:
+      payload:
+        $ref: '#/components/schemas/temperatureId'
   schemas:
     temperatureId:
       type: object
       additionalProperties: false
       properties:
         temperatureId:
           type: string
-```
```

Comment:
are you sure this should be removed? did you check generated preview? isn't it breaking the docs UI?